python/fastfec.py

```diff
@@ -1,59 +1,386 @@
-import sys, getopt
-from ctypes import *
+"""
+A Python library to interface with FastFEC.
+
+Import the library helper via:
+
+    from fastfec import FastFEC
+
+Receive an instance of the LibFastFEC class that automatically handles freeing up used memory:
+
+    with FastFEC() as fastfec:
+        ...
+
+Example usage:
+
+    * Parse a filing line by line:
+
+        from fastfec import FastFEC
+
+        with open('12345.fec', 'rb') as f:
+            with FastFEC() as fastfec:
+                for form, line in fastfec.parse(f):
+                    print("GOT", form, line)
+
+    * Parse a filing to output files:
+
+        import os
+        from pathlib import Path
+        from fastfec import FastFEC
+
+        # Custom open method that places files in an output directory,
+        # and creates the directory if it does not exist
+        def open_output_file(filename, *args, **kwargs):
+            filename = os.path.join('custom_python_output', filename)
+            output_file = Path(filename)
+            output_file.parent.mkdir(exist_ok=True, parents=True)
+            return open(filename, *args, **kwargs)
+
+        with open('12345.fec', 'rb') as f:
+            with FastFEC() as fastfec:
+                fastfec.parse_as_files(f, open_output_file)
+"""
```

Comment:
I mean, yes. Pylint is just mad I didn't have a doc comment. I will add a README eventually and port this over

Reply:
IMO we should have some version of a readme for this sooner rather than later. It can be a different PR if we need it to be, but we should be shipping some basic examples for how we expect people to use this.

Reply:
Added the README